meetings/2019-03-f2f.md

```diff
@@ -0,0 +1,247 @@
+# AMP F2F 2019-03-27/28 in London
+
+**Agenda:** https://github.com/ampproject/meta-ac/issues/22
+
+## Day 1
+
+#### Attendees:
+
+* Terence Eden @edent
+* Tobie Langel @tobie
+* Charles Vazac @cvazac
+* Joe Alicata @wirelessjoe
+* Levi Durfee @levidurfee
+* Léonie Watson @ljwatson
+
+### What is the TSC
+
+* Technical Steering Committee.
+* Inspired by NodeJS style terminology.
```

Comment:
It is inspired by NodeJS style terminology.

Reply:
OK, I'll only fix spelling errors (without getting in the British vs. US english debate). They're notes, so I'm fine with quasi sentences and the like.